src/CommunityToolkit.Maui.Core/Handlers/Popup/PopupHandler.tizen.cs

```diff
@@ -0,0 +1,96 @@
+﻿using CommunityToolkit.Maui.Core.Views;
+using Tizen.UIExtensions.NUI;
+
+namespace CommunityToolkit.Maui.Core.Handlers;
+
+public partial class PopupHandler : Microsoft.Maui.Handlers.ElementHandler<IPopup, MauiPopup>
+{
+	/// <inheritdoc/>
+	protected override MauiPopup CreatePlatformElement() => new MauiPopup(MauiContext ?? throw new NullReferenceException(nameof(MauiContext)));
+
+	/// <summary>
+	/// Action that's triggered when the Popup is closed.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	/// <param name="result">The result that should return from this Popup.</param>
+	public static void MapOnClosed(PopupHandler handler, IPopup view, object? result)
+	{
+		var popup = handler.PlatformView;
+
+		if (popup.IsOpen)
+		{
+			popup.Close();
+		}
+
+		handler.DisconnectHandler(popup);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup is Opened.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	/// <param name="result">We don't need to provide the result parameter here.</param>
+	public static void MapOnOpened(PopupHandler handler, IPopup view, object? result)
+	{
+		handler.PlatformView.ShowPopup();
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup is dismissed by tapping outside of the Popup.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	/// <param name="result">The result that should return from this Popup.</param>
+	public static void MapOnDismissedByTappingOutsideOfPopup(PopupHandler handler, IPopup view, object? result)
+	{
+		if (view.CanBeDismissedByTappingOutsideOfPopup)
+		{
+			view.OnDismissedByTappingOutsideOfPopup();
+		}
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup <see cref="IPopup.Anchor"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	public static void MapAnchor(PopupHandler handler, IPopup view)
+	{
```

Comment:
what about these and others `Map Handlers` that don't have implementation?? Is because there's no support for tizen or something is missing?

Reply:
Anchor was applied when popup is opened and we not support changing anchor during a opened popup. so no need anything here. 
I will add some comment to intent empty code